examples/policies/README.md

```diff
@@ -0,0 +1,31 @@
+# Policies examples
+
+You can start example configuration with example policy by runing this from the APIcast root directory:
+
+```shell
+bin/apicast -c examples/policies/example.json -b --policy-load-path examples/policies
+```
+
+And then make a request by:
+
+```shell
+curl 'localhost:8080/test'
+```
+
+And see a response:
+
+```http
+GET /test HTTP/1.1
+X-Real-IP: 127.0.0.1
+Host: echo
+User-Agent: curl/7.57.0
+Accept: */*
+Example: Value
+```
+
+And see similar line in the APIcast log:
+
+```
+2018/02/08 10:01:48 [warn] 72249#10132050: *9 [lua] init.lua:19: op(): setting header: Example to: Value, client: 127.0.0.1, server: _, request: "GET /test HTTP/1.1", host: "localhost:8080"
```

Comment:
This shouldn't be a warning, no? It's the purpose of the policy. Warning sounds like something is not working as it should, I'd use a different log level for the example.

Reply:
Ok. Will tune it to info/notice level.

Reply:
Done.